include/clang/Driver/Options.td

```diff
@@ -2024,6 +2024,9 @@ def mglibc : Flag<["-"], "mglibc">, Group<m_libc_Group>, Flags<[HelpHidden]>;
 def muclibc : Flag<["-"], "muclibc">, Group<m_libc_Group>, Flags<[HelpHidden]>;
 def module_file_info : Flag<["-"], "module-file-info">, Flags<[DriverOption,CC1Option]>, Group<Action_Group>,
   HelpText<"Provide information about a particular module file">;
+def msandbox : Flag<["-"], "msandbox">, Group<m_Group>,
+  HelpText<"CHERI Pure Capability ABI generated code will be run in a sandbox (CHERI only)">,
```

Comment:
I have fixed this in 86f9eaa. Should we also temporarily output a compiler error if someone specifies -msandbox without -mabi=purecap?